src/api.js

```diff
@@ -15,16 +17,29 @@ window.api = {
   doExport() {
     return window.application.$store.getters['exportData'];
   },
-  setConfig({
-    units = 'm',
-    showMapDialogOnStart = true,
-    online = true,
-    onChange = function change() {},
-  }) {
-    this.config = { units, showMapDialogOnStart, online, onChange, };
+  setConfig(config) {
+    if (this.setConfigAlreadyRun) {
+      throw new Error('This method can only be run once!');
+    }
+    if (config === undefined) { config = {}; }
+    this.config = Object.assign({
+      units: 'm',
+      showMapDialogOnStart: true,
+      online: true,
+      onChange: function change() { window.versionNumber += 1; },
```

Comment:
I think it'd be good to make this an anonymous function (remove `change` from `function change()`). Otherwise, we might clobber some variable of the same name.